v8/payloads/channel.ts

```diff
@@ -154,6 +155,23 @@ export enum MessageFlags {
 	URGENT = 1 << 4,
 }
 
+export interface APIMessageSticker {
```

Comment:
let's call it `APISticker` instead (who knows maybe in the future they'll release..profile stickers, sticker stickers, etc)